test/signin.test.js

```diff
@@ -117,6 +115,7 @@ describe('POST /susers/sessions', () => {
             })
             .then(response => {
               expect(response.status).toBe(200);
+              expect(response.body.token.length).toBeGreaterThan(0);
```

Comment:
Ideally you would test that the token de-encrypts well and that you can obtain whats inside it. But I know that should be coming in the next PR so you can change it later.